index.html

```diff
@@ -154,7 +177,7 @@ <h2>A project for first-time contributions</h2>
                 <p>3 Useful Dev Resources</p>
                 <ul>
                     <li>
-                        <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">FreeCodeCamp</a>
+                        <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">Udemy</a>
```

Comment:
```suggestion
                        <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">FreeCodeCamp</a>
```

you shouldn't edit other's card